plugins/modules/proxmox_backup_info.py

```diff
@@ -0,0 +1,198 @@
+    api_token_id: "{{ token_id | default(omit) }}"
+    api_token_secret: "{{ token_secret | default(omit) }}"
+    validate_certs: "{{ validate_certs | default(omit) }}"
+  register: proxmox_backups
+'''
+RETURN = '''
+proxmox_backups:
+  description: List of backup job.
+  returned: on success
+  type: list
+  elements: dict
+  contains:
+    mailnotification:
+      description: Notification event type.
+      returned: on success
+      type: str
+    storage:
+      description: Storage used in backup job.
+      returned: on success
+      type: str
+    notes-template:
+      description: Template string for generating notes for the backup(s).
+      returned: on success
+      type: str
+    mailto:
+      description: Comma-separated list of email addresses or users that should receive email notifications.
+      returned: on success
+      type: str
+    id:
+      description: Job ID.
+      returned: on success
+      type: str
+    mode:
+      description: Backup mode.
+      returned: on success
+      type: str
+    next-run:
+      description: Timestamp of next run.
+      returned: on success
+      type: str
+    repeat-missed:
+      description: If true, the job will be run as soon as possible if it was missed while the scheduler was not running.
+      returned: on success
+      type: bool
+    type:
+      description: Type of backup.
+      returned: on success
+      type: str
+    enabled:
+      description: If true, this job is enable else it is disable the job.
+      returned: on success
+      type: bool
+    all:
+      description: If true, this job bacukup all known guest systems on this host.
+      returned: on success
+      type: bool
+    pool:
+      description: Backup mode.
+      returned: on success
+      type: str
+    vmid:
+      description: The ID of the guest system backuped.
+      returned: on success
+      type: str
+'''
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.urls import fetch_url
+from ansible_collections.community.general.plugins.module_utils.proxmox import (
+    proxmox_auth_argument_spec, ProxmoxAnsible, proxmox_to_ansible_bool)
+
+
+class ProxmoxBackupInfoAnsible(ProxmoxAnsible):
+    def __init__(self, module):
+        self.module = module
+        self.api_host = module.params['api_host']
+        self.api_user = module.params['api_user']
+        self.api_password = module.params.get('api_password')
+        self.api_token_id = module.params.get('api_token_id')
+        self.api_token_secret = module.params.get('api_token_secret')
+        self.validate_certs = module.params['validate_certs']
+
+    def fetch_backup_data(self, endpoint):
+        headers = {
+            'Authorization': 'PVEAPIToken={0}!{1}={2}'.format(self.api_user, self.api_token_id, self.api_token_secret)
+        }
+        url = 'https://{0}:8006/api2/json/{1}'.format(self.api_host, endpoint)
+        response, info = fetch_url(self.module, url, headers=headers)
```

Comment:
Shouldn't this module use proxmoxer (instead of doing direct HTTP calls), as all (or at least most of) the other proxmox_* modules also do?